app/ts/components/subcomponents/solidityType.tsx

```diff
@@ -0,0 +1,34 @@
+import { EnrichedGroupedSolidityType, PureGroupedSolidityType } from '../../types/solidityType.js'
+import { RenameAddressCallBack } from '../../types/user-interface-types.js'
+import { checksummedAddress, dataStringWith0xStart } from '../../utils/bigint.js'
+import { assertNever } from '../../utils/typescript.js'
+import { SmallAddress } from './address.js'
+
+const textStyle = 'text-overflow: ellipsis; overflow: hidden;'
+const StringElement = ({ text }: { text: string }) => <p class = 'paragraph' style = { textStyle }>{ `${ text }` }</p>
```

Comment:
text is already a string here

```suggestion
const StringElement = ({ text }: { text: string }) => <p class = 'paragraph' style = { textStyle }>{ text }</p>
```